checkov/terraform/tf_parser.py

```diff
@@ -391,6 +453,29 @@ def parse_hcl_module_from_tf_definitions(
                     logging.warning(e, exc_info=False)
         return module, tf_definitions
 
+    def parse_hcl_module_from_multi_tf_definitions(
+        self,
+        tf_definitions: list[dict[TFDefinitionKey, dict[str, Any]]],
+        source_dir: str,
+        source: str,
+    ) -> tuple[Module, list[dict[TFDefinitionKey, dict[str, Any]]]]:
+        module = self.get_new_module(
+            source_dir=source_dir,
+            module_address_map=self.module_address_map,
+            external_modules_source_map=self.external_modules_source_map,
+        )
+        self.add_tfvars_with_source_dir(module, source, source_dir)
+        copy_of_tf_definitions = pickle_deepcopy(tf_definitions)
+        for tf_def in copy_of_tf_definitions:
+            for file_path, blocks in tf_def.items():
+                for block_type in blocks:
+                    try:
+                        module.add_blocks(block_type, blocks[block_type], file_path, source)
+                    except Exception as e:
+                        logging.warning(f'Failed to add block {blocks[block_type]}. Error:')
+                        logging.warning(e, exc_info=False)
```

Comment:
just thinking about the log level, if maybe `info` is enough, depends on how critical it is to the user.

Reply:
Since it shouldn't happen normally I'm OK with warning, this way we can also search for it in our logs more easily. It's not critical enough to log error in my opinion.
Also, in the original function it's logged as warning as well.